packages/require/lib/index.ts

```diff
@@ -0,0 +1,143 @@
+import path from "path";
+import TruffleConfig from "@truffle/config";
+import { options as expectOptions } from "@truffle/expect";
+import { Web3Shim, createInterfaceAdapter } from "@truffle/interface-adapter";
+
+import vm from "vm";
+import Module from "module";
+
+import originalRequire from "original-require";
+
+import { ExecOptions, RequireOptions } from "./types";
+import { compile } from "./typescript";
+
+import Debug from "debug";
+
+const debug = Debug("@truffle/require:index");
```

Comment:
I don't think I've seen the debug string prefixed by `@ truffle`, it usually starts with the package name, like  `dashboard-message-bus-client:connection`
```suggestion
const debug = Debug("require:index");
```